src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1387,17 +1403,26 @@ private function dumpValue($value, $interpolate = true)
 
             return sprintf('array(%s)', implode(', ', $code));
         } elseif ($value instanceof IteratorArgument) {
+            $countCode = array();
+            $countCode[] = 'function() {';
```

Comment:
I did first. But any function generated in the dumped container by the PhpDumper gives `function()` currently.